src/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -258,6 +263,7 @@ const CreateScenarioDialog = ({
           </Grid>
           <Grid item xs={12}>
             <Autocomplete
+              data-cy="scenario-type-select"
```

Comment:
Could be interesting to prefix the id by  `create-scenario-dialog` in order to narrow the possibilities to have several field with the same data-cy id?
Something like  `create-scenario-dialog-type-select`